tests/integration/utils.py

```diff
@@ -180,6 +181,30 @@ def escape_path_go(dependency):
         return dependency
 
 
+def get_hash_from_file(filename):
```

Comment:
Should the function name include the algorithm used (sha256)? Alternatively, it could receive a parameter with the algorithm requested.